projects/serializers/common.py

```diff
@@ -146,7 +146,7 @@ def serialize_projects(
 
     # sort by order to allow any prioritized tags to be shown first
     # e.g. global leaderboard tags
-    for obj in sorted(projects, key=lambda x: x.order, reverse=True):
+    for obj in sorted(projects, key=lambda x: x.order or float("inf")):
```

Comment:
`Order` field has default value as `0`. Let's replace `or float("inf")` with `sorted(..., reverse=True)` to make it more clear

Reply:
Unfortunately, we want the order to be:
`[1, 2, 3, ..., 9, 0, 0, 0, 0]`
At least, that's the way it's used for things like the topics ordering in the sidebar, so I was avoiding changing convention.